tests/testthat/test-util.R

```diff
@@ -95,3 +95,20 @@ test_that("write_lines recovers on error", {
   expect_silent(write_lines(value, filename))
   expect_identical(readLines(filename), value)
 })
+
+
+test_that("read_text_file() works", {
```

Comment:
Test for:

* error on missing file
* error on empty file
* read only first line of multiline file(!)

Reply:
See 1f3c05eb0ec4eb299a5f8c337e754a41966bad98. For multi-line files, `read_text_file("file_name", too_many_characters)` includes `'\n'`. Is that okay? For key files, we are going to know how many characters we want, and it would take a little overhead to sub out `'\n'`.